### PR TITLE
Validates email#1633

### PR DESCRIPTION
**Pull request by bijoysijo**
Fixes:
solves #1633 
Fixes username validation;
with conditions:
- Username can't be blank.

Fixes email validation;
with condition:
- Validates email field to check if a domain name is present after the '@' in email, while a user signs up.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -31,7 +31,12 @@ class User < ApplicationRecord
   after_commit :create_members_from_invitations, on: :create
 
   has_attached_file :profile_picture, styles: { medium: "205X240#", thumb: "100x100>" }, default_url: ":style/Default.jpg"
+
+  # validations for user
+
   validates_attachment_content_type :profile_picture, content_type: %r{\Aimage/.*\z}
+  validates :name , presence: true
```
[the remark this answers is not in the document]
Comment (bijoysijo) [reply app/models/user.rb]: resolved

File: spec/system/signup_spec.rb [hunk 2]
```diff
@@ -13,9 +13,8 @@
 
   it "should not sign-up when no credentials" do
     click_button "Sign up"
-
-    expect(page).to have_text("Email can't be blank")
-    expect(page).to have_text("Password can't be blank")
+
```
[the remark this answers is not in the document]
Comment (bijoysijo) [reply spec/system/signup_spec.rb]: resolved

**Comment by satu0king** [comment]
Thanks!
This is similar to #1649, @pavanjoshi914 can you check this.

**Comment by pavanjoshi914** [comment]
@satu0king  yes, makes username required and regex for email but not through devise!

**Pull request merged**